PDFTron/webviewer-ui
# Update importAnnotations and exportAnnotations to use promise

Manual tested:

- Call `readerControl.downloadPdf()` in the `documentLoaded` event listener
- Print in Chrome

This change should be merged into dev immediately after [this PR](https://github.com/XodoDocs/webviewer/pull/67) is merged
